content/cli/kes-key/decrypt.md

```diff
@@ -8,42 +8,34 @@ tableOfContents: true
 
 ## Overview
 
-Decrypt the contents of a message with a key.
+Decrypt the contents of a data encryption key with a key and return the plaintext key.
+
+Use the plaintext value for encrypting or decrypting data using the DEK. 
+
+{{< admonition type="caution">}}
+Avoid storing the plaintext on disk, as it allows decryption of data without requiring access to the Secret key used to generate the DEK.
```

Comment:
Perhaps `plaintext value` to be consistent with the previous sentence? Or possibly `plaintext key` for both.

Reply:
Taking first suggestion.